Havit.Blazor.Documentation/Pages/Components/HxMultiSelectDoc/HxMultiSelect_Demo_BasicUsage.razor

```diff
@@ -4,6 +4,7 @@
                TItem="EmployeeDto"
                TValue="int"
                Data="@employees"
+               LabelType="LabelType.Floating"
```

Comment:
Do not modify the basic demo. Please revert.
If you need to test the implementation, use the new `TestApp` or the soon-to-be-removed `BlazorAppTest`

Reply:
Most input components don’t have separate demos for floating labels, as the basic idea is introduced here:  
https://havit.blazor.eu/components/Inputs#floating-labels  
(Which reminds me - we should update the list of inputs supporting floating labels on that page.)  

If you think the `HxMultiSelect` is unique in how it handles floating labels and it’s worth presenting separately, feel free to do so. I’ll leave that decision up to you.